pipeline/dbt/models/intermediate/sources/fredo/int_fredo__services.sql

```diff
@@ -0,0 +1,193 @@
+        CAST(NULL AS TEXT [])                                                                 AS "pre_requis",
+        CAST(NULL AS BOOLEAN)                                                                 AS "cumulable",
+        CAST(NULL AS TEXT [])                                                                 AS "justificatifs",
+        CAST(NULL AS DATE)                                                                    AS "date_creation",
+        CAST(NULL AS DATE)                                                                    AS "date_maj",
+        CAST(NULL AS DATE)                                                                    AS "date_suspension",
+        CAST(NULL AS BOOLEAN)                                                                 AS "contact_public",
+        CAST(NULL AS TEXT [])                                                                 AS "modes_accueil",
+        LEFT(structures.presentation_resume, 280)                                             AS "presentation_resume",
+        CASE
+            WHEN CHAR_LENGTH(structures.presentation_resume) > 280
+                THEN structures.presentation_resume
+        END                                                                                   AS "presentation_detail",
+        ARRAY[(
+            SELECT di_nom_and_types_by_fredo_services.di_type
+            FROM di_nom_and_types_by_fredo_services
+            WHERE services.value = di_nom_and_types_by_fredo_services.service_fredo
+        )]                                                                                    AS "types",
+        (
+            SELECT REPLACE((UPPER(SUBSTRING(di_nom_and_types_by_fredo_services.service_fredo, 1, 1)) || SUBSTRING(di_nom_and_types_by_fredo_services.service_fredo, 2)), '/', 'et')
+            FROM di_nom_and_types_by_fredo_services
+            WHERE services.value = di_nom_and_types_by_fredo_services.service_fredo
+        )                                                                                     AS "nom",
+        CASE
+            WHEN frais.frais IS NULL THEN frais_autres.frais_autres
+        END                                                                                   AS "frais_autres",
+        ARRAY['envoyer-un-mail', 'telephoner']                                                AS "modes_orientation_accompagnateur",
+        COALESCE(mode_orient_benef.mode_orient_benef, ARRAY['envoyer-un-mail', 'telephoner']) AS "modes_orientation_beneficiaire"
+    FROM structures
+    INNER JOIN services ON structures.id = services.structure_id
+    LEFT JOIN thematiques ON structures.id = thematiques.structure_id
+    LEFT JOIN profils ON structures.id = profils.structure_id
+    LEFT JOIN frais ON structures.id = frais.structure_id
+    LEFT JOIN frais_autres ON structures.id = frais_autres.structure_id
+    LEFT JOIN mode_orient_benef ON structures.id = mode_orient_benef.structure_id
```

Comment:
nit

mentalement j'aurais plutot fait dans ce sens, pour s'assurer qu'on a bien la bonne quantité d'éléments. Ca joue des tours parfois, cela dit ça n'a pas d'impact ici !

```suggestion
    FROM services
    LEFT JOIN structures ON services.structure_id = structures.id
    LEFT JOIN thematiques ON structures.id = thematiques.structure_id
    LEFT JOIN profils ON structures.id = profils.structure_id
    LEFT JOIN frais ON structures.id = frais.structure_id
    LEFT JOIN frais_autres ON structures.id = frais_autres.structure_id
    LEFT JOIN mode_orient_benef ON structures.id = mode_orient_benef.structure_id
```